fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -358,6 +359,7 @@ SECOND: 'SECOND';
 SECURITY: 'SECURITY';
 SELECT: 'SELECT';
 SEMI: 'SEMI';
+SEPARATOR: 'SEPARATOR';
```

Comment:
plz add these two words `GROUP_CONCAT` and `SEPARATOR` to nonReserved in `Starrocks.g4`

Reply:
added